frontend/app/src/components/wrapped/WrappedConfetti.vue

```diff
@@ -0,0 +1,171 @@
+<script setup lang="ts">
+interface ConfettiParticle {
+  color: string;
+  rotation: number;
+  rotationSpeed: number;
+  size: number;
+  velocityX: number;
+  velocityY: number;
+  x: number;
+  y: number;
+}
+
+// Composable: Use this to manage confetti animation
+function useConfetti() {
+  // Refs
+  const confettiCanvas = ref<HTMLCanvasElement | null>(null);
+  const ctx = ref<CanvasRenderingContext2D | null>(null);
```

Comment:
is there any reason we can't do this here?
```suggestion
  const confettiCanvas = ref<HTMLCanvasElement>();
  const ctx = ref<CanvasRenderingContext2D>();
```